src/components/draw/feature-edit-style-symbole.vue

```diff
@@ -0,0 +1,45 @@
+<script setup lang="ts">
+import { Ref, inject } from 'vue'
+import { useTranslation } from 'i18next-vue'
+
+// import { DrawFeature } from '@/stores/draw.store.model' // TODO:
+import Circle from '@/components/common/symbol/circleSymbol.vue'
+import Rectangle from '@/components/common/symbol/rectangleSymbol.vue'
+import Cross from '@/components/common/symbol/crossSymbol.vue'
+import Triangle from '@/components/common/symbol/triangleSymbol.vue'
+
+type FeatureShape = 'circle' | 'square' | 'cross' | 'triangle'
+
+const { t } = useTranslation()
+const currentEditCompKey: Ref<'FeatureEditSymbol' | undefined> | undefined =
+  inject('currentEditCompKey')
+// const feature: DrawFeature | undefined = inject('feature') // TODO:
+const featureShape: FeatureShape = 'circle' // feature.olFeature.get('shape') // TODO: to plug when feature ok
+const featureColor = 'red' // feature.olFeature.get('color') // TODO: to plug when feature ok
```

Comment:
This will be part of the logical part anyway, but did I understand right, that the problem of using `DrawnFeature.featureStyle.color` here is, that it currently needs a `DrawnFeature.changed()` call for `ol` to update it?

Reply:
Exactly, we should improve Drawnfeature class to avoid calling ol functionalities in our components.